tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,38 @@
+@api
+@replication @replication-aditions
+Feature: Contentpool media replication basically works.
+
+  Make sure that media referenced in content can be pulled from remote contentpool server.
+
+  @javascript
+  Scenario: Replication media works
+    Given I am logged in to contentpool
+    When I visit path "node/add/article" on contentpool
+    # Choose Cooking, since satellite wants to replicate articles with Food channel and their descendants.
+    And I fill in "title[0][value]" with "BEHAT: Media"
+    And I fill in "field_seo_title[0][value]" with "test media"
+    And I select "-Cooking" from "edit-field-channel"
+    And I select "published" from "moderation_state[0]"
+
+    #Teaser
+    And I press "field_teaser_media_entity_browser_entity_browser"
+    Then I wait for AJAX to finish
+    Then I wait for ".views-row:nth-of-type(2)" in entity browser "image_browser"
+    When I click on ".views-row:nth-of-type(2)" in entity browser "image_browser"
+    Then ".views-row:nth-of-type(2)" in entity browser "image_browser" should have the class "checked"
+    And I click on "#edit-submit" in entity browser "image_browser"
+    And I wait for entity browser "image_browser" to close
+    Then I wait for AJAX to finish
+    Then "[name='field_teaser_media[target_id]']" value should be "media:3"
```

Comment:
Seems to duplicate existing step:
>Then the element $locator should contain $foo ?

?

    /**
     * Checks, that element with specified CSS contains specified HTML
     * Example: Then the "body" element should contain "style=\"color:black;\""
     * Example: And the "body" element should contain "style=\"color:black;\""
     *
     * @Then /^the "(?P<element>[^"]*)" element should contain "(?P<value>(?:[^"]|\\")*)"$/
     */
    public function assertElementContains($element, $value)